packages/zipkin-transport-http/src/HttpLogger.js

```diff
@@ -9,7 +9,9 @@ class HttpLogger {
     const timer = setInterval(() => {
       this.processQueue();
     }, httpInterval);
-    timer.unref();
+    if (timer.unref) {
+      timer.unref();
```

Comment:
@jquatier ps what's the impact of "not doing this" in javascript that doesn't support it? I should make a code comment I think.